commit/src/pcs.rs

```diff
@@ -64,20 +64,35 @@ where
         evaluations: impl IntoIterator<Item = (Self::Domain, RowMajorMatrix<Val<Self::Domain>>)>,
     ) -> (Self::Commitment, Self::ProverData);
 
-    /// Commit to the quotient polynomials. If `zk` is not enabled, this is the same as `commit`.
-    /// If `zk` is enabled, the quotient polynomials are randomized as explained in Section 4.2 of
-    /// https://eprint.iacr.org/2024/1037.pdf .
+    /// Commit to the quotient polynomial. We first decompose the quotient polynomial into
+    /// `decomposition_degree` many smaller polynomials each of degree `degree / decomposition_degree`.
+    /// This can have minor performance benefits, but is not strictly necessary in the non `zk` case.
+    /// When `zk` is enabled, this commitment will additionally include some randomization process
+    /// to hide the inputs.
     ///
-    /// *** Arguments
-    /// - `domains` are the domains of the quotient polynomial chunks we need to commit to.
-    /// - `evaluations` are the evaluations of the quotient polynomial chunks we need to commit to.
+    /// ### Arguments
+    /// - `quotient_domain` the domain of the quotient polynomial.
+    /// - `quotient_evaluations` the evaluations of the quotient polynomial over the domain. This should be in
+    ///   standard (not bit-reversed) order.
+    /// - `decomposition_degree` the number of smaller polynomials to decompose the quotient polynomial into.
     #[allow(clippy::type_complexity)]
     fn commit_quotient(
         &self,
-        domains: Vec<Self::Domain>,
-        evaluations: Vec<RowMajorMatrix<Val<Self::Domain>>>,
+        quotient_domain: Self::Domain,
+        quotient_evaluations: RowMajorMatrix<Val<Self::Domain>>,
+        decomposition_degree: usize,
```

Comment:
I find this name a bit confusing without reading the doc comment. Maybe we can just call it `num_chunks`?